frontend/src/metabase-types/store/dashboard.ts

```diff
@@ -56,7 +56,7 @@ export interface DashboardState {
   parameterValues: Record<ParameterId, ParameterValueOrArray>;
 
   loadingDashCards: {
-    dashcardIds: DashCardId[];
+    dashboardId: number | null;
```

Comment:
This is a new state, used for checking if `loadingIds` are for the dashboard we're loading. This is because this PR introduce a way for dashboard to still have ongoing `loadingIds` while loading other dashcards that aren't in `loadingIds`.